pipelines/HyperSuprimeCam/ApVerifyWithFakes.yaml

```diff
@@ -5,6 +5,6 @@ instrument: lsst.obs.subaru.HyperSuprimeCam
 description: Fully instrumented AP pipeline with fakes, specialized for HSC
 imports:
```

Comment:
Why does the DECam version of this pipeline have a whole separate `processVisitFakes` section, but this HSC one does not?

Reply:
Good question... those configs were added on https://github.com/lsst/ap_verify/commit/801f904a44fde924046e860e977dcd5debc8dad9 without explanation. I think they might have been cargo-culted from the `calibrate` config in https://github.com/lsst/ap_pipe/commit/1c8f5877dee5610a62ffc02ffa25072d70dab025. There is no analogous set of configs for `calibrate` in the HSC `ProcessCcd.yaml`.

Reply:
Oh, I see, OK.

Reply:
Nevermind, per above DECam comment, there are no special characterize or calibrate configs in HSC's ProcessCcd pipeline, so we don't need to have any config overrides here either.